Week3/README.md

```diff
@@ -1,20 +1,14 @@
 # Reading material for the fourth lecture:
```

Comment:
I think we should bundle `call`, `apply` and `bind` with the discussion of `this` which I think should be in Week 7 as part of OOP. I don't think any of these methods is required in homework before week 7. In fact, of the three we have limited use for `bind` only (in the React module, but even then we could avoid it with fat arrow functions assigned to class properties).